arch/inst/Zimop/mop.rr.n.yaml

```diff
@@ -3,9 +3,11 @@
 $schema: "inst_schema.json#"
 kind: instruction
 name: mop.rr.n
-long_name: No synopsis available.
+long_name: May be an OPeration (2 registers).
 description: |
-  No description available.
+  The Zimop extension defines 8 MOP instructions named MOP.RR.n, where n is an integer between 0
+  and 7, inclusive. Unless redefined by another extension, these instructions simply write 0 to X[rd].
+  Their encoding allows future extensions to define them to read X[rs1] and X[rs2], as well as write X[rd].
```

Comment:
Similarly, if this single definition eventually gets expanded to 32 instruction definitions, then this text should probably change to something like:
> These instructions simply write 0 to X[rd].
> 
> May-be-operations (MOPs) are designed to be redefined by later extensions to perform some other action. The encoding allows future extensions to define it to read X[rs1] and X[rs2], as well as write X[rd].